templates/gcp/index.html

```diff
@@ -0,0 +1,189 @@
+          <li class="p-list__item is-ticked">Updates mirrored and images published in all GCP regions worldwide</li>
+          <li class="p-list__item is-ticked">5-year lifetime</li>
+        </ul>
+        <p>Run Ubuntu 18.04 LTS: Standard (Coming soon)</p>
+      </div>
+    </div>
+    <div class="col-6 p-card">
+      <h3 class="p-card__title">Ubuntu Pro for GCP</h3>
+      <p class="p-heading--two">Launching soon - 3-4.5% of instance cost</p>
+      <hr class="u-sv1">
+      <div class="p-card__content">
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">Includes all Ubuntu optimisations and updates of the free version</li>
+          <li class="p-list__item is-ticked">Expanded security covering 26,000 additional application components delivered with Ubuntu</li>
+          <li class="p-list__item is-ticked">Kernel live patching for instant security and longer uptimes</li>
+          <li class="p-list__item is-ticked">FIPS 140-2 and CC-EAL certified components</li>
+          <li class="p-list__item is-ticked">Integration with GCP security and compliance features</li>
+          <li class="p-list__item is-ticked">10-year lifetime</li>
+        </ul>
+        <p><a href="/gcp/pro">More about Ubuntu Pro for GCP&nbsp;&rsaquo;</a></p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-12">
+      <h2>Features</h2>
+    </div>
+  </div>
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Optimised</h3>
+      <p>Ubuntu on GCP runs on an GCP-optimised kernel, which includes improved device drivers, and out of the box support for accelerators like GPUs. This means faster instance starts and better runtime performance for your workloads.</p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Security built in</h3>
+      <p>Our security teams track alerts 24x7 and release patches to system components and applications continuously. Canonical operates a worldwide distribution network ensuring that package updates are delivered in-region quickly. Canonical also works with Google to provide the latest <a href="https://cloud.google.com/blog/products/identity-security/introducing-google-cloud-confidential-computing-with-confidential-vms">Confidential Computing</a> technology in Cloud.</p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">All the bits you need</h3>
+      <p>Ubuntu includes everything you need out of the box to build and operate cloud-native applications at scale. Supporting runtimes like Python, Node.js and Go, and server components like Apache Kafka, NGINX, MongoDB, Redis and PostgreSQL.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
+      <h2>Get Ubuntu Pro on GCP (coming soon):</h2>
```

Comment:
```suggestion
      <h2>Get Ubuntu Pro on GCP <span class="p-label--updated" style="position:relative;top: -1em;">Coming soon</span></h2>
```

Reply:
Done. it doesn't seem to be vertically aligned, I changed it to  `-0.5em` instead

Reply:
I thought we would want this as a superscript style so high but see what design think.